src/Pages/ResetPassword/ResetPassword.tsx

```diff
@@ -1,10 +1,142 @@
+    <>
+      <Box component="div">
+        {" "}
+        <Typography variant="h4" component="h4" sx={{ padding: "20px" }}>
+          <Box component="span" sx={{ color: "skyblue" }}>
+            Stay
+          </Box>
+          cation.
+        </Typography>
+      </Box>
+      <Box sx={{ padding: "30px 70px" }}>
+        <Box component="div">
+          <Typography variant="h4" component="h4">
+            Reset Password
+          </Typography>
+          <Typography>
+            If you already have an account register <br /> You can
+            <Link to="/login">Login here !</Link>
+          </Typography>
+        </Box>
+      </Box>
+      <form
+        onSubmit={handleSubmit(onSubmit)}
+        style={{ width: "80%", margin: "auto" }}
+      >
+        <InputLabel sx={{ marginTop: "5px" }}>Email</InputLabel>
+        <FormControl sx={{ width: "100%", margin: "20px 0" }}>
+          <OutlinedInput
+            type="email"
+            placeholder="Please type here ..."
+            {...register("email", {
+              required: true,
+              pattern: /^[^@]+@[^@]+\.[^@ .]{2,}$/,
+            })}
+          />
+          {errors.email && errors.email.type === "required" && (
+            <Box className="text-danger" sx={{ color: "red" }}>
+              email is required
+            </Box>
+          )}
+          {errors.email && errors.email.type === "pattern" && (
+            <Box component="span" sx={{ color: "red" }}>
+              invalid email
+            </Box>
+          )}
+        </FormControl>
+        <InputLabel sx={{ marginTop: "5px" }}>OTP</InputLabel>
+        <FormControl sx={{ width: "100%", margin: "20px 0" }}>
+          <OutlinedInput
+            type="text"
+            placeholder="Please type here ..."
+            {...register("seed", {
+              required: true,
+            })}
+          />
+          {errors.seed && errors.seed.type === "required" && (
+            <Box className="text-danger" sx={{ color: "red" }}>
+              OTP is required
+            </Box>
+          )}
+        </FormControl>
+        <InputLabel sx={{ marginTop: "5px" }}>Password</InputLabel>
+        <FormControl sx={{ width: "100%", margin: "20px 0" }}>
+          <OutlinedInput
+            type="password"
+            placeholder="Please type here ..."
+            {...register("password", {
+              required: true,
+            })}
+          />
+          {errors.password && errors.password.type === "required" && (
+            <Box className="text-danger" sx={{ color: "red" }}>
+              password is required
+            </Box>
+          )}
+        </FormControl>
+        <InputLabel sx={{ marginTop: "5px" }}>Confirm Password</InputLabel>
+        <FormControl sx={{ width: "100%", margin: "20px 0" }}>
+          <OutlinedInput
+            type="password"
+            placeholder="Please type here ..."
+            {...register("confirmPassword", {
+              required: true,
```

Comment:
you might use the validate method from react-hook-form, to check whether the new password matches the confirmPassword or not, and if not, return `Passowrds don't match!`